tools/github/scripts/Snapshot-AzsdkLabels.ps1

```diff
@@ -0,0 +1,150 @@
+[CmdletBinding(DefaultParameterSetName = 'RepositoryFile', SupportsShouldProcess = $true)]
+param (
+    [Parameter()]
+    [ValidateScript({Test-Path $_ -PathType 'Container'})]
+    [string]$SnapshotDirectory = "$PSScriptRoot/../data/repository-snapshots",
+
+    [Parameter()]
+    [ValidateScript({Test-Path $_ -PathType 'Leaf'})]
+    [string] $LabelsFilePath = "$PSScriptRoot/../data/common-labels.csv",
+
+    [Parameter(ParameterSetName = 'RepositoryFile')]
+    [ValidateScript({Test-Path $_ -PathType 'Leaf'})]
+    [string] $RepositoryFilePath = "$PSScriptRoot/../data/repositories.txt",
+
+    [Parameter(ParameterSetName = 'Repositories')]
+    [ValidateNotNullOrEmpty()]
+    [string[]] $Repositories = @(
+        'Azure/azure-sdk-for-cpp'
+        'Azure/azure-sdk-for-go'
+        'Azure/azure-sdk-for-java'
+        'Azure/azure-sdk-for-js'
+        'Azure/azure-sdk-for-net'
+        'Azure/azure-sdk-for-python'
+        'Azure/azure-sdk-tools'
+    ),
+
+    [Parameter(ParameterSetName = 'Languages')]
+    [ValidateNotNullOrEmpty()]
+    [string[]] $Languages = @('cpp', 'go', 'java', 'js', 'net', 'python'),
```

Comment:
If you keep these do you not want to add android, ios, and c? We should try to keep those 9 languages in-sync.

Reply:
No objection, though we should do that consistently across all the scripts.   I'll work that in.